CHANGELOG.md

```diff
@@ -5,17 +5,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unpublished Version / DEV]
 
-- [#237](https://github.com/nf-core/scrnaseq/pull/237) Add `singularity.registry = 'quay.io'` and bump NF version to 23.04.0
+## v2.4.0 - 2023-08-16 Lime Platinum Crab
+
+- Fix typo causing empty version imformation for mtx_conversion subworkflow ([#254](https://github.com/nf-core/scrnaseq/pull/254))
+- Add `singularity.registry = 'quay.io'` and bump NF version to 23.04.0 ([#237](https://github.com/nf-core/scrnaseq/pull/237))
 - Fixed issue with file collisions while using cellranger ([#232](https://github.com/nf-core/scrnaseq/pull/232))
 - Fix issue where multiqc inputs tried to access objects that did not exist ([#239](https://github.com/nf-core/scrnaseq/pull/239))
-- Removed `public_aws_ecr` profile
+- Removed `public_aws_ecr` profile ([#242](https://github.com/nf-core/scrnaseq/pull/242))
 - Include cellranger in MultiQC report ([#244](https://github.com/nf-core/scrnaseq/pull/244))
 - Nf-core template update to v2.9 ([#245](https://github.com/nf-core/scrnaseq/pull/245))
 - Update cellranger and fastqc module ([#246](https://github.com/nf-core/scrnaseq/pull/246)).
   The [updated cellranger module](https://github.com/nf-core/modules/pull/3537) now automatically renames input FASTQ
   files to match the expected naming conventions.
 
-## v2.3.2 - 2023-06-07 Patched Yellow Strontium Pinscher
+## v2.3.2 - 2023-06-07 Sepia Samarium Salmon
```

Comment:
what happened there?

Reply:
The change log was inconsistent with the release name on github. I updated the changelog to match the release.